src/foam/u2/ActionView.js

```diff
@@ -220,6 +220,11 @@ foam.CLASS({
       padding: 7px 15px;
     }
 
+    ^tertiary^small {
```

Comment:
Why `^tertiary^small` and not `^tertiary-small`.  Using a caret to separate "tertiary" and "small" will expand into a CSS class name of

`foam-u2-ActionView-tertiary-foam-u2-ActionView-small`, seems a little redundant :)

Reply:
Hey @adamvy , since the caret includes the class selector '.' it actually comes out as a chained two classes on an element rather than one combined class!
`.foam-u2-ActionView-tertiary.foam-u2-ActionView-small`

Reply:
Gotcha, I misunderstood that the caret also introduces the period character.